nebula-logger/tests/logger/classes/LogEntryEventHandler_Tests.cls

```diff
@@ -5,7 +5,241 @@
+            LimitsMobilePushApexCallsUsed__c  = 123,
+            LimitsQueueableJobsMax__c         = 987,
+            LimitsQueueableJobsUsed__c        = 123,
+            LimitsSoqlQueriesMax__c           = 987,
+            LimitsSoqlQueriesUsed__c          = 123,
+            LimitsSoqlQueryLocatorRowsMax__c  = 987,
+            LimitsSoqlQueryLocatorRowsUsed__c = 123,
+            LimitsSoqlQueryRowsMax__c         = 987,
+            LimitsSoqlQueryRowsUsed__c        = 123,
+            LimitsSoslSearchesMax__c          = 987,
+            LimitsSoslSearchesUsed__c         = 123,
+            Locale__c                         = 'es-ES',
+            LoggingLevel__c                   = 'DEBUG',
+            LoggingLevelOrdinal__c            = 9,
+            LoginDomain__c                    = 'https://fake.my.salesforce.com',
+            LoginHistoryId__c                 = '255.255.255.255',
+            LoginType__c                      = null,
+            LogoutUrl__c                      = null,
+            Message__c                        = 'My message',
+            MessageTruncated__c               = false,
+            NetworkId__c                      = 'Fake-ID',
+            OriginLocation__c                 = 'right here',
+            OriginType__c                     = 'Apex',
+            ProfileId__c                      = currentUser.ProfileId,
+            RelatedRecordId__c                = currentUser.Id,
+            RelatedRecordJson__c              = Json.serializePretty(currentUser),
+            SessionId__c                      = null,
+            SessionSecurityLevel__c           = null,
+            SessionType__c                    = null,
+            SourceIp__c                       = null,
+            StackTrace__c                     = 'I started here \nand then went there',
+            SystemModeIsApexRest__c           = false,
+            SystemModeIsBatch__c              = false,
+            SystemModeIsFuture__c             = false,
+            SystemModeIsQueueable__c          = false,
+            SystemModeIsScheduledJob__c       = false,
+            Timestamp__c                      = System.now(),
+            TimeZoneId__c                     = 'fake-id',
+            TimeZoneName__c                   = 'My time zone name',
+            TransactionId__c                  = transactionId,
+            Type__c                           = 'Diagnostic',
+            UserLoggingLevel__c               = 'DEBUG',
+            UserLoggingLevelOrdinal__c        = 9,
+            UserRoleId__c                     = currentUser.UserRoleId,
+            UserType__c                       = null
+        );
+
+        Test.startTest();
+        Database.SaveResult saveResult = EventBus.publish(logEntryEvent);
+        Test.getEventBus().deliver();
+
+        System.assertEquals(true, saveResult.isSuccess(), saveResult.getErrors());
+
+        Log__c log = getLog();
```

Comment:
@jamessimone for some reason, this line for `getLog()` always fails in my tests because it doesn't seem like the events are getting published (line 230 above). I tried a few different options, but I can't get it working - am I missing something obvious?

Here's an example of the test failure, in case that helps - https://travis-ci.org/github/jongpie/NebulaLogger/builds/735648328